ledger/participant-state/kvutils/tools/BUILD.bazel

```diff
@@ -74,6 +74,7 @@ da_scala_test(
         "//ledger/participant-state",
         "//ledger/participant-state/kvutils",
         "//ledger/participant-state/kvutils/tools",
+        "@maven//:io_grpc_grpc_api",
```

Comment:
Can we please sort the deps?

Reply:
done, hopefully in the way you are preferring it.